src/containers/containerCollection/containerCollection.jsx

```diff
@@ -75,7 +75,7 @@ export class ContainerCollection extends Component {
         })
     }
 
-    handleEditContainerEnterKey () {
+    handleEditContainerEnterKey (e) {
```

Comment:
Can you just change this to the event itself, and call e.key inside the if statement.